lib/ex_doc/markdown/earmark.ex

```diff
@@ -74,6 +76,44 @@ defmodule ExDoc.Markdown.Earmark do
     "$$\n#{content}\n$$"
   end
 
+  # Convert admonition blockquotes to divs for screen reader accessibility
+  defp fixup(
+         {"blockquote", blockquote_attrs, [{tag, h_attrs, _h_content, _h_meta} = h_elem | rest] = ast,
+          blockquote_meta}
+       )
+       when tag in ["h3", "h4"] do
+    h_admonition_classes =
+      h_attrs
+      |> Enum.find("", &match?({"class", _}, &1))
+      |> then(fn
+        "" ->
+          ""
+
+        {"class", classes} ->
+          classes
+          |> String.split(" ")
+          |> Enum.filter(&(&1 in @admonition_classes))
+          |> Enum.join(" ")
+      end)
```

Comment:
I tried to keep everything self-contained instead of adding helper functions, so that's why this is a little messy. The "coercion" to `""` is there since the call to `Enum.join/2` will return an empty string for an empty list.